### DISCUSSION TITLE
How to prohibit manual kubectl edit or apply for some resources in the cluster?

### DISCUSSION THREAD

**Discussion by andrii-korotkov-verkada** [argoproj/argo-cd#17489] [Q&A] [answered]
Hello everyone,
I hope your day is going well. 
I want to set up a new ArgoCD application with ConfigMaps, synced from a specific folder in Github repository. In addition, I want to make non-admin users not be able to manually edit those ConfigMaps or apply arbitrary changes, so only changes from Github master are synced. Do you know how it can be done and whether it's configurable on ArgoCD/application level somewhere?
Thanks!

**Comment by vorozhko** [comment, marked as answer]
You can use Application auto sync feature, so any changes will be automatically detected and reverted back by argodcd controller.

**Reply by mosheavni** [reply]
but if I have server side diff? it would see that the controller of those fields is `kubectl-edit` and would ignore those changes by default.
